tests/PHPStan/Analyser/nsrt/bug-13197.php

```diff
@@ -0,0 +1,42 @@
+<?php declare(strict_types = 1);
+
+namespace Bug13197;
+
+use function PHPStan\Testing\assertType;
+
+/**
+ * @return array{string, string}|null STDOUT & STDERR tuple
+ */
+function execute(string $command): ?array
+{
+	if (!function_exists('proc_open')) {
+		return null;
+	}
+
+	$pipes = [];
+
+	$process = @proc_open(
+		$command,
+		[
+			['pipe', 'rb'],
+			['pipe', 'wb'], // stdout
+			['pipe', 'wb'], // stderr
```

Comment:
```suggestion
			['pipe', 'wb'], // stderr
			4 => ['pipe', 'wb'], // https://stackoverflow.com/questions/28909347/is-it-possible-to-connect-more-than-the-two-standard-streams-to-a-terminal-in-li#28909376
```

to shown/test the result can be also not a `list` and `@template` the keys is helpful.